sqlglot/expressions.py

```diff
@@ -1105,6 +1105,14 @@ def ctes(self):
             return []
         return with_.expressions
 
+    @property
+    def selects(self):
+        raise NotImplementedError("Subqueryable objects must implement `selects`")
```

Comment:
Which expressions don't implement this?
Should we just implement them?

Reply:
They all do now (I just merged in a change that implemented it in union). I put this in place now to make it clear that future expressions should implement.